### DISCUSSION TITLE
Quickly run command/script for a game

### DISCUSSION THREAD

**Discussion by IcyAlmond** [lutris/lutris#3905] [Ideas]
Ability to quickly run a custom user-defined script or command on a game through right clicking and pressing a button.

**Comment by mihawk90** [comment]
Can you go into more detail for this?

This is already possible. See for example the Overwatch installer and a couple others.

**Reply by IcyAlmond** [reply]
For example an update script for a game. 
Or a script which clears the dxvk cache due if the game commonly has problems with it.

(manually added scripts by the user, or added by the initial installer script

**Reply by mihawk90** [reply]
As I said this is already possible. One major example is the Overwatch installer. In the GUI it's the "Manual Script" option in the System options tab of a game.
